arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -325,6 +325,76 @@
             "description": "OK"
           }
         }
+      },
+      "get": {
```

Comment:
@amarzavery Yes, we've received feedback from people, saying the methods that take in parentResourcePath is not at all intuitive. It is pretty cumbersome to construct the request URI for the above using parentResourcePath. A simpler implementation would be the one suggested by you (similar to [this](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-resources/policy/2016-04-01/swagger/policy.json#L38)) or a path that points to a a fully qualified /{lockId}